### PR TITLE
fix: implements binary comparison operator for bool

### PR DESCRIPTION
**Pull request by baszalmstra**
Implements missing binary comparison operator

### CODE REVIEW

**Comment by Wodann** [comment]
Is there a reason not to include the other comparative operators?

**Comment by baszalmstra** [comment]
> Is there a reason not to include the other comparative operators?
I dont feel like its intuitive, is `true` > `false`?

**Comment by Wodann** [comment]
> I dont feel like its intuitive, is `true` > `false`?

Yes. At first I thought it might be a useless operator, but on second thought I realised those operators are necessary when ordering `bool` or when making an algorithm that allows transitions from `false -> false`, `false -> true`, and `true -> true` when running a loop, but no "reversions" (i.e. `true -> false`)

File: crates/mun_codegen/src/test.rs [hunk 1]
```diff
@@ -125,6 +125,17 @@ fn assignment_op_bool() {
     );
 }
 
+#[test]
```
Comment (Wodann) [inline crates/mun_codegen/src/test.rs:128]: This test is no longer necessary

**Pull request merged**